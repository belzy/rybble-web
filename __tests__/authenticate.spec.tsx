import { render } from '@testing-library/react';
import Authenticate from '../pages/authenticate';
import '@testing-library/jest-dom';

describe('pages/authenticate', () => {

  it('should render', () => {

    const { container } = render(<Authenticate />);

    expect(container).toMatchSnapshot();

  });

});