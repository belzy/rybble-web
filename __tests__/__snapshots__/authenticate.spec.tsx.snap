// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/authenticate should render 1`] = `
<div>
  <div
    class="css-qy22cp"
  >
    <div
      class="css-j7qwjs"
    />
    <div
      class="css-j7qwjs"
    />
  </div>
</div>
`;
